src/plugins/data_source_management/public/components/cluster_selector/cluster_selector.tsx

```diff
@@ -16,7 +16,7 @@ export const LocalCluster: ClusterOption = {
   id: '',
 };
 
-interface ClusterSelectorProps {
+export interface ClusterSelectorProps {
```

Comment:
```suggestion
export interface DataSourceSelectorProps {
```

Reply:
Cut another issue to address the naming https://github.com/opensearch-project/OpenSearch-Dashboards/issues/6038